setup.py

```diff
@@ -103,7 +103,7 @@ def build_extension(self, ext):
 
 kwargs = {}
 
-version = "5.1"
+version = "5.2"
```

Comment:
It would be great if the version is something like `5.2j` because that is the naming convention for our forks